cluster.go

```diff
@@ -97,18 +92,6 @@ func NewCluster(connStr string, credential Credential, opts ...*ClusterOptions)
 		connectTimeout = duration
 	}
 
-	if valStr, ok := fetchOption("timeout.dispatch_timeout"); ok {
-		duration, err := time.ParseDuration(valStr)
-		if err != nil {
-			return nil, invalidArgumentError{
-				ArgumentName: "timeout.dispatch_timeout",
-				Reason:       err.Error(),
-			}
-		}
-
-		dispatchTimeout = duration
-	}
-
 	if valStr, ok := fetchOption("timeout.uery_timeout"); ok {
```

Comment:
Not related to this PR, but I just noticed the typo here. Should be `timeout.query_timeout`